EssentialApp/EssentialAppTests/FeedUIIntegrationTests.swift

```diff
@@ -317,11 +317,31 @@ final class FeedUIIntegrationTests: XCTestCase {
 		wait(for: [exp], timeout: 1.0)
 	}
 	
+	func test_feedImageView_imageSelectionTapExecutesImageSelectionBlock() {
+		let image0 = makeImage(url: URL(string: "http://url-0.com")!)
+		let image1 = makeImage(url: URL(string: "http://url-1.com")!)
+		
+		var selectedImages: [FeedImage] = []
+		let (sut, loader) = makeSUT { selectedImages.append($0) }
+		
+		sut.loadViewIfNeeded()
+		loader.completeFeedLoading(with: [image0, image1])
```

Comment:
Missing pre-condition assertion to increase coverage.

```suggestion
		sut.loadViewIfNeeded()
		loader.completeFeedLoading(with: [image0, image1])
		XCTAssertEqual(selectedImages, [])
```